composer/algorithms/layer_freezing/layer_freezing.py

```diff
@@ -22,8 +22,8 @@ def freeze_layers(
     model: Model,
     optimizers: Optimizers,
     current_duration: float,
-    freeze_start: float,
-    freeze_level: float,
+    freeze_start: float = 0.5,
+    freeze_level: float = 1.0,
```

Comment:
Does the corresponding `hparams` need to be updated?

Reply:
Nope, they already use these defaults